test/petab/test_petab_import.py

```diff
@@ -76,6 +76,21 @@ def test_3_optimize(self):
             self.assertTrue(np.isfinite(
                 result.optimize_result.get_for_key('fval')[0]))
 
+    def test_check_gradients(self):
+        # check gradients of simple model (should always be a true positive)
+        model_name = "Bachmann_MSB2011"
+        petab_problem = pypesto.petab.PetabImporter.from_yaml(
+            os.path.join(folder_base, model_name, model_name + '.yaml'))
+
+        objective = petab_problem.create_objective()
+        objective.amici_solver.setSensitivityMethod(
+                 amici.SensitivityMethod_forward)
+        objective.amici_solver.setAbsoluteTolerance(1e-10)
+        objective.amici_solver.setRelativeTolerance(1e-12)
+
+        self.assertFalse(petab_problem.check_gradients(
```

Comment:
why is this False again? maybe add a scenario where it's true? this https://github.com/ICB-DCM/pyPESTO/blob/develop/test/util.py#L122 should be stable enough no matter what.